app/controllers/account/my_tenants_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Account::MyTenantsController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @tenant = current_user.tenant
+  end
+
+  def edit
+    @tenant = current_user.tenant
+  end
+
+  def update
+    @tenant = current_user.tenant
```

Comment:
```show```, ```edit``` and ```update``` has this line ```@tenant = current_user.tenant``` so probably it would be good idea to move it to ```before_action``` in order to follow DRY.

Reply:
Yeah, it's really a good idea, thanks!) Done.